config/routes.rb

```diff
@@ -32,6 +32,7 @@
   resources :posts, only: %i(show destroy) do
     member do
       get :interactions
+      get :mentionable_in_comment
```

Comment:
I'm unsure about it. Since it also needs a `post_id` as parameter it should be clear that it means mentionable in the context of the post with this id. But if you think it's ambiguous I think `PostService#mentionable_in_comment` is also OK, or you can add a documentation for the method.